### ISSUE TITLE
Unable to get suggestions without typing a specific order of parameters

### ISSUE THREAD

**Issue by mhalano** [redhat-developer/vscode-yaml#705]
## Describe the bug
I'm trying to get suggestions to kind parameter. I want to get kind = Pod just typing "Po" and ctrl-space, but it says "no suggestions". I'm using apiVersion = v1. If I already type other root parameters, the kind parameter completes. Like, if I type metadata parameter and the kind, it works. If I start with kind (after apiVersion, of course) doesn't work. This is a problem because I use kind always a second root parameter.

## Expected Behavior

Kind parameter should suggest "Pod" in the auto-complete when you type "Po".

## Current Behavior

Appears "No suggestions" instead the "Pod" suggestion

## Steps to Reproduce

1. Start typing a small YAML file:
```
apiVersion: v1
kind: Po
```
2. Try to auto-complete to get "Pod". It will get "No suggestions" instead.
3. If you already typed another root parameter, should work:
```
apiVersion: v1
metadata:
  name: test
kind: Po
```
4. Try to auto-complete to get "Pod". It will get "Pod" as a suggestion.

## Environment
- [ ] Windows
- [ ] Mac
- [X] Linux
- [ ] other (please specify)

**Comment by gorkem** [comment]
How do you map the YAML file to Kubernetes in your setup? This extension does not automatically map YAML files to be Kubernetes. Usually, either user configures the mapping or uses some [extension](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools) that does this mapping.

**Comment by mhalano** [comment]
@gorkem I have this snippet in my settings.json:
```
"yaml.schemas": {
    "kubernetes": "*.yaml"
}
```

**Comment by mhalano** [comment]
@gorkem Were you able to reproduce my bug?

**Comment by gorkem** [comment]
Yes, I was able to reproduce.

**Comment by hefayed** [comment]
i am having same problem, not sure if you found a solution for this ?

**Comment by mhalano** [comment]
Do we have any news about that?

**Comment by msivasubramaniaan** [comment]
@mhalano This issue is resolved on the v1.8.0. Please check and confirm

**Comment by mhalano** [comment]
@msivasubramaniaan I was able to test and confirmed the version 1.8.0 is showing the suggestions correctly. Thank you. 

@hefayed if you update the extension it should be work now.

I'm closing this bug since it's solved. Thank you all.
